src/main/flight/pid.c

```diff
@@ -114,7 +114,7 @@ PG_RESET_TEMPLATE(pidConfig_t, pidConfig,
 
 #define LAUNCH_CONTROL_YAW_ITERM_LIMIT 50 // yaw iterm windup limit when launch mode is "FULL" (all axes)
 
-PG_REGISTER_ARRAY_WITH_RESET_FN(pidProfile_t, MAX_PROFILE_COUNT, pidProfiles, PG_PID_PROFILE, 6);
+PG_REGISTER_ARRAY_WITH_RESET_FN(pidProfile_t, MAX_PROFILE_COUNT, pidProfiles, PG_PID_PROFILE, 7);
```

Comment:
When just appending elements on to the PG it's not necessary to increment the version.

Reply:
At which linearization value? Maybe the value was chosen too high resulting in over-compensation?

Reply:
@etracer65, it's clear that this feature increases pid authority and noise at low rpm. Pid need to be retuned and the linearization factor also needs to be selected appropriately as @markusgritsch reports. So the oscillations are expected and just need to be managed.

Note that the feature is off by default. This allows our users to experiment with it without creating undue risk. First impressions have been fairly positive.

Reply:
@ledvinap Ahh yes, forgot that it was an array. Ignore my comment on the PG version increment.